stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -17,16 +17,23 @@ def check_qty(self):
             if record.quants_lines:
                 move = self.env['stock.move'].browse(
                     self.env.context['active_id'])
-                if record.lines_qty > move.product_uom_qty:
+                move_qty = self.env['product.uom']._compute_qty_obj(
+                    move.product_uom, move.product_uom_qty,
+                    move.product_id.uom_id)
```

Comment:
You could use the `move.product_qty` field, which already does this :
https://github.com/odoo/odoo/blob/c8ca71fc62b44dbb6ba9eebf9091dae87c723cf4/addons/stock/stock.py#L1840